src/engine/app-manager.ts

```diff
@@ -111,57 +112,57 @@ export abstract class AppManagerApi implements API<AppManager> {
 
   /** Activate a module or plugin */
   public activateOne(name: string) {
-    const entry = this.getEntity(name)
-    this.activateCallAndEvent(entry)
-    if (this.isPlugin(entry)) {
-      this.activateRequestAndNotification(entry)
+    const api = this.getEntity(name)
+    this.activateCallAndEvent(api)
+    if (this.isPlugin(api)) {
+      this.activateRequestAndNotification(api)
     }
-    if (entry.api.activate) {
-      entry.api.activate()
+    if (api.activate) {
+      api.activate()
     }
     this.setActive(name, true)
-    this.events.emit('activate', entry)
+    this.events.emit('activate', api)
   }
 
   /** Activation for Module and Plugin */
-  private activateCallAndEvent<T extends Api>({ profile, api }: Entry<T>) {
-    const events = profile.events || []
-    events.forEach(event => {
+  private activateCallAndEvent<T extends Api>(api: PluginApi<T>) {
+    const events = api.profile.events || []

```

Comment:
should be `api.events` ?

Reply:
`api.events` : the event emitter
`api.profile.events` : the names of the events exposed